webui/react/src/components/TaskList.tsx

```diff
@@ -596,7 +595,7 @@ const TaskList: React.FC<Props> = ({ workspace }: Props) => {
     }: {
       children: React.ReactNode;
       onVisibleChange?: (visible: boolean) => void;
-      record: AnyTask;
+      record: CommandTask;
```

Comment:
is it ok to replace  `AnyTask` with `CommandTask` here? the `record` can be `AnyTask`, right?

Reply:
this component only deals in CommandTasks, record comes from ` useState<CommandTask[]` which is fetching NTSC tasks